content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -36,8 +37,12 @@ In the RBAC API, a role contains rules that represent a set of permissions.
 Permissions are purely additive (there are no "deny" rules).
 A role can be defined within a namespace with a `Role`, or cluster-wide with a `ClusterRole`.
 
-A `Role` can only be used to grant access to resources within a single namespace.
-Here's an example `Role` in the "default" namespace that can be used to grant read access to pods:
+A Role sets permissions within a {{< glossary_tooltip text="namespace" term_id="namespace" >}},
+whereas a ClusterRole applies across all namespaces in your
+{{< glossary_tooltip text="cluster" term_id="cluster" >}}.
+
+Here's an example Role that grants access to read all the
```

Comment:
The previous text was more accurate. The role "can be used to grant read access" (when used in combination with a rolebinding), but does not grant that access on its own.